bookkeeper-server/src/main/java/org/apache/bookkeeper/conf/ServerConfiguration.java

```diff
@@ -431,6 +432,27 @@ public ServerConfiguration setVerifyMetadataOnGc(boolean verifyMetadataOnGC) {
         return this;
     }
 
+    /**
+     * Get zk-operation timeout in seconds used by GC while performing zk
```

Comment:
If this is only for GC we should provide a more meaningful name.
Can you change it please?

Reply:
i think this could be generic configuration for timeout and can be used at anywhere where bookie needs blocking zk call. However, I think we can also use existing `zkTimeout` which can genuinely guarantees that zk-session is alive and callback couldn't complete. or let me know if you guys think we need different name?